index.js

```diff
@@ -89,7 +89,8 @@ function start(opts) {
   }
 
   if (!stats && !loghose && !dockerEvents) {
-    throw new Error('you should enable at least one of stats, logs or dockerEvents');
+    throw new Error(`You should enable either stats, logs or dockerEvents, \
+this might be due to missing log token.`);
```

Comment:
Can it be because of that? I don't think these are related to the log token?

Reply:
It is although it doesn't look like it, expand the diff above, if `logs`, `stats` or `eventToken` aren't set, `stats`, `loghose` and `dockerEvents` might not be set, this will trigger if either you exclude all logging or don't provide a token